api/activity.go

```diff
@@ -53,6 +53,11 @@ const (
 	ActivityProjectMemberDelete ActivityType = "bb.project.member.delete"
 	// ActivityProjectMemberRoleUpdate is the type for updating project member roles.
 	ActivityProjectMemberRoleUpdate ActivityType = "bb.project.member.role.update"
+
+	// SQL(Editor) related
+
+	// ActivitySQLQueryHistory is the type for executing query.
+	ActivitySQLQueryHistory ActivityType = "bb.sql.query.history"
```

Comment:
bb.sql-editor.query

there is no need to mention history. "Query" here is used as a verb since we are recording an action.